app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,7 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="colorBoulder">#787575</color>
+    <color name="colorFlushMahogany">#CF473A</color>
+    <color name="colorScorpion">#575757</color>
```

Comment:
You can use descriptive names on the colours, related to the place where they are used to keep consistency
```suggestion
    <color name="button_background_tint">#787575</color>
    <color name="op_button_background_tint">#CF473A</color>
```

Reply:
Color constant names have been changed to be more descriptive.